client/my-sites/stats/stats-purchase/stats-purchase-commercial.tsx

```diff
@@ -49,15 +51,21 @@ const CommercialPurchase = ( { planValue }: CommercialPurchaseProps ) => {
 					`By clicking the button below, you agree to our {{a}}Terms of Service{{/a}} and to {{b}}share details{{/b}} with WordPress.com.`,
 					{
 						components: {
-							a: <Button variant="link" href="#" />,
+							a: (
+								<Button
+									variant="link"
+									target="_blank"
+									href={ localizeUrl( 'https://wordpress.com/tos/' ) }
+								/>
+							),
 							b: <Button variant="link" href="#" />,
 						},
 					}
 				) }
 			</p>
 
 			<Button variant="primary">
-				{ translate( 'Get Jetpack Stats for %(value)s per month', {
+				{ translate( 'Get Jetpack Stats for $%(value)s per month', {
```

Comment:
Just curious, is the subscription price always in USD?

Reply:
Thanks for pointing this out. I will add currency translation to use the user's currency.